dev-python/pyside/pyside-9999.ebuild

```diff
@@ -135,8 +142,27 @@ src_configure() {
 
 	configuration() {
 		local mycmakeargs=(
+			-DPYTHON_EXECUTABLE="${PYTHON}"
```

Comment:
**Wild.** On my system, passing both `-DPYTHON_EXECUTABLE` and `-DPYTHON_EXTENSION_SUFFIX` suffices to find the current Python 3.x interpreter.

Admittedly, my system's probably on drugs. <sup>_Every time._</sup>

Not passing `-DPYTHON_EXECUTABLE` seems like the wrong thing to do. Or does CMake have some other means of determining the current Python 3.x interpreter? Perhaps I have no idea what I'm doing.